examples/reach.amp.html

```diff
@@ -0,0 +1,60 @@
+<!doctype html>
```

Comment:
Add this file to the example-validation test (and whitelist the error there).

You may also consider just implementing the validation changes yourself. Otherwise file a ticket for us.